lib/miio/Status.js

```diff
@@ -4,16 +4,17 @@
  */
 const Status = function() {
     /** List of allowed values as per Roborock.STATUS_MAP */
-    this.state = "UNKNOWN";
+    this.state = 0;
```

Comment:
I think using strings or an enum here is strictly better. And roborock updateStatus also sets this to a string.

Reply:
That was only a fix due to type checking.
There is Code which uses it as number.

Reply:
The one in webserver.js right? Should fix those locations instead.

Reply:
It is used almost everywhere as a number, even your fakerobot.js sets it to 2.
Maybe it should be refactored later, it's not the topic of this pull request.

Reply:
fakerobot sends a json object not a Status one.
It is converted by updateStatus to a string: https://github.com/Hypfer/Valetudo/blob/viomi/lib/devices/Roborock.js#L173
The remaining usage in webserver.js as an int is a bug that should be fixed.
But not necessarily here. Feel free to merge and we can fix it later.